packages/core/src/html/linkProcessor.js

```diff
@@ -87,16 +87,41 @@ function convertMdAndMbdExtToHtmlExt(node) {
     if (hasNoConvert) {
       return;
     }
+
     const { href } = node.attribs;
-    const hasMdExtension = href.slice(-3) === '.md';
+
+    const hasMdExtension = href.includes('.md');
     if (hasMdExtension) {
```

Comment:
Would having a guard clause for `hasMdExtension` and `hasURLFragment` be better so as to reduce nesting?

Reply:
Yup, that's a good suggestion! I think would make the different cases clearer as well. Will work on it 👍